src/components/UserMenu/UserMenuPresentation.js

```diff
@@ -105,33 +105,36 @@ function UserMenuPresentation({ currentUser, login, logout }) {
               <Menu.Item className="w-full flex">
                 {({ active }) => (
                   // eslint-disable-next-line jsx-a11y/anchor-is-valid, jsx-a11y/no-static-element-interactions
-                  <a type="button" onClick={login} onKeyDown={login}>
+                  <button type="button" onClick={login} onKeyDown={login}>
                     <div
                       className={`${
                         active ? menuItemActiveClass : menuItemInactiveClass
-                      } ${menuItemBaseClass}`}
+                      } ${menuItemBaseClass} flex justify-start`}
                     >
                       Sign In / Register
                     </div>
-                  </a>
+                  </button>
                 )}
               </Menu.Item>
             ) : (
               <Menu.Item className="w-full flex">
                 {({ active }) => (
                   // eslint-disable-next-line jsx-a11y/anchor-is-valid, jsx-a11y/no-static-element-interactions
-                  <a type="button" onClick={logout} onKeyDown={logout}>
+                  <button type="button" onClick={logout} onKeyDown={logout}>
                     <div
                       className={`${
                         active ? menuItemActiveClass : menuItemInactiveClass
-                      } ${menuItemBaseClass}`}
+                      } ${menuItemBaseClass} flex justify-start`}
                     >
                       Sign out
                     </div>
-                  </a>
+                  </button>
                 )}
               </Menu.Item>
             )}
+            <Menu.Item className="text-fv-charcoal px-2 py-1 w-full flex text-lg whitespace-nowrap font-medium hover:bg-gray-200 rounded">
```

Comment:
If you wrap the link in an arrow function like the other menu items and use the same classes it will keep the styling the same and mean we only have to maintain the styling in one place e.g. `className={`${ active ? menuItemActiveClass : menuItemInactiveClass} ${menuItemBaseClass}`}`.

See the docs for extra info: https://headlessui.com/react/menu